Tests/fixtures/tv-4.yml

```diff
@@ -4989,3 +4989,57 @@
     model: CT-8750
   os_family: Android
   browser_family: Unknown
+-
+  user_agent: Mozilla/5.0 (X11; Linux aarch64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/100.0.4896.127 Odin/100.4896.4.10 Safari/537.36 Model/VIDAA-NT72690 VIDAA/7.0(Smarttech;SmartTV;VS0343FE2;NT72690/V0000.07.00C.O0629;FHD;VS0343F2;)
+  os:
+    name: VIDAA
+    version: "7.0"
+    platform: ARM
+  client:
+    type: browser
+    name: Odin
+    version: 100.4896.4.10
+    engine: Blink
+    engine_version: 100.0.4896.127
+  device:
+    type: tv
+    brand: Smart Tech
+    model: ""
+  os_family: GNU/Linux
+  browser_family: Chrome
+-
+  user_agent: Mozilla/5.0 (X11; Linux x86_64;) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/47.0.2526.106 Safari/537.36 SRAF/4.0 (; Hisense; SmartTV_2019_1; V0000.01.00a.N0423; MICALIDVB6886; ); CE-HTML/1.0 hbbtv
+  os:
+    name: GNU/Linux
+    version: ""
+    platform: x64
+  client:
+    type: browser
+    name: Seraphic Sraf
+    version: "4.0"
+    engine: Blink
+    engine_version: 47.0.2526.106
+  device:
+    type: tv
+    brand: Hisense
+    model: Smart TV (2019)
+  os_family: GNU/Linux
+  browser_family: Chrome
+-
+  user_agent: Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/73.0.3683.103 Safari/537.36 SRAF/5.0 1.0 (MediaTek; MT5870; V0.1.0.0.M001; ) NETRANGEMMH
+  os:
+    name: GNU/Linux
+    version: ""
+    platform: x64
+  client:
+    type: browser
+    name: Seraphic Sraf
+    version: "5.0"
+    engine: Blink
+    engine_version: 73.0.3683.103
+  device:
+    type: tv
+    brand: MediaTek
+    model: MT5870
```

Comment:
This is most likely the processor model, Also, mediatek does not produce televisions, only processor chips

Reply:
I know, but we already have it here - https://github.com/matomo-org/device-detector/blob/7c0853dc2bbd37544f73d68290718fe64ac1fa94/regexes/device/televisions.yml#L659-L665.

I just added it again in mobiles.yml otherwise it couldn't be detected.

Reply:
I removed MediaTek